mautic/mautic
# Trackable link to mailto

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [No]
| New feature/enhancement? (use the a.x branch)      | [Yes]
| Deprecations?                          | [No]
| BC breaks? (use the c.x branch)        | [No]
| Automated tests included?              | [No] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

The enhancement allows to track a link using the mailto protocol in the body of an email.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create Email with mailto link in the body of email
3. Send Email by Campaign
4. Click mailto link in the body of email
5. Check Click Counts link in the Email view
6. Please note that links are not tracked while you are logged in to the Mautic dashboard.
 
![image-2022-09-14-16-42-12-429](https://user-images.githubusercontent.com/100592709/190669386-ea4586f1-de3f-4708-ab1f-bdbae3864e0b.png)
